frontend/src/routes/app/(reader)/[collection_slug]/[publication_slug]/+page.svelte

```diff
@@ -0,0 +1,42 @@
+<script lang="ts">
+    import { onMount } from 'svelte';
+    import { page } from '$app/stores';
+    import api from '$lib/api';
+    import { 
+        items, 
+        loadingItems, 
+        publications, 
+        selectedPublication 
+    } from '../../../appStore';
+
+    let lastFetchedSlug: string | null = null;
+
+    $effect(() => {
```

Comment:
- make the route optional `[[publication_slug]]`
- if slug is not set, load collection_items